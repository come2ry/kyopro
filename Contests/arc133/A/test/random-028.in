10
3 1 5 11 8 11 11 4 2 4
