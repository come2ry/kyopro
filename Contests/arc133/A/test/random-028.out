1 5 1 1 8 1 1 1 1 4 2 4
